packages/app/src/app/pages/NewDashboard/Components/Selection/ContextMenu.tsx

```diff
@@ -496,6 +559,48 @@ const MultiMenu = ({ selectedItems }: IMultiMenuProps) => {
   );
 };
 
+const MasterMenu = ({
+  repo,
+}: {
+  repo: {
+    owner: string;
+    name: string;
+    branch: string;
+  };
+}) => {
+  const { actions } = useOvermind();
+  const { visible, setVisibility, position } = React.useContext(Context);
+
+  return (
+    <Menu.ContextMenu
+      visible={visible}
+      setVisibility={setVisibility}
+      position={position}
+      style={{ width: 120 }}
+    >
+      <MenuItem
+        onSelect={() =>
+          window.open(
+            `https://codesandbox.io/s/github/${repo.owner}/${repo.name}`
```

Comment:
Same here, is there a better way that we can generate the URL?

Reply:
I think it would be better to use relative URLs as well (so it works on staging for example).

Reply:
done :)